homeassistant/components/landisgyr_heat_meter/sensor.py

```diff
@@ -81,6 +81,15 @@ class HeatMeterSensorEntityDescription(
         state_class=SensorStateClass.TOTAL,
         value_fn=lambda res: res.heat_usage_gj,
     ),
+    HeatMeterSensorEntityDescription(
```

Comment:
This is an integration change, and directly required/related to the addition of the unit made. This should not be in the same PR.

Reply:
Oh, i am sorry, i was thinking that it is also need to be changed, removing this currently